### ISSUE TITLE
Script /rc.d/unifi.sh

### ISSUE THREAD

**Issue by hacesoft** [unofficial-unifi/unifi-pfsense#283]
Have a nice day, the script of which is at:

https://github.com/unofficial-unifi/unifi-pfsense/blob/72ce5f92cb978e26027d952f2d71fc133a8e21d1/rc.d/unifi.sh

is for creating a menu directly in PfSense? Possible switching on and off unifi controllers directly conveniently from the firewall menu instead of uncomfortably from the console?

**Comment by vtermaat** [comment]
Would be nice to be able to see:
1. If the service and dependencies are running:
- The Unifi Controller
-  Java
- Mongodb

2. Output of the startup script, which could help identify issues.

F.e. after PfSense 2.6.0 upgrade I had to go through [these steps](https://github.com/unofficial-unifi/unifi-pfsense#java-compatibility-on-freebsd) because Java wouldn't run. And because I didn't know how to view the output of the startup script I was basically flying blind.

**Comment by hacesoft** [comment]
Have a nice day,
so the script is part of the main installation block of the controller :). When I looked at the code in more detail, I found a link to it. Although I don't exactly understand the code, I don't know its exact syntax, but I guess I understand.

I was already glad that it is a help code that I will see in the PfSense GUI whether the controller service is installed and started / stopped ...
